charts/alfresco-search-enterprise/values.yaml

```diff
@@ -88,14 +88,24 @@ reindexing:
         limits:
           cpu: "0.25"
           memory: "10Mi"
+ats:
+  # -- URL of the alfresco transform (trouter or tengine-aio)
+  transform_url: null
+  # -- URL of the alfresco shared filestore (trouter or tengine-aio)
+  sfs_url: null
+  existingConfigMap:
+    # -- Alternatively, provide ATS details via an existing configmap
+    name: null
+    keys:
+      # -- Key within the configmap holding the URL of the alfresco transform
+      transform_url: ATS_URL
+      # -- Key within the configmap holding the URL of the alfresco shared filestore
+      sfs_url: SFS_URL
 elasticsearch:
   # -- Enable embedded elasticsearch - useful when using this chart in standalone
   enabled: false
   replicas: 1
   clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
```

Comment:
do you want to move also this to the bottom if the intention is to have all values related to testing at the bottom?